src/test/java/io/github/robwin/swagger/SwaggerConsumerDrivenAssertTest.java

```diff
@@ -140,6 +140,15 @@ public void shouldHandleDefinitionsUsingAllOfIncludingCycles() {
             .satisfiesContract(designFirstSwaggerLocation.getAbsolutePath());
     }
 
+    @Test
+    public void shouldntFailWhenDesignHasNoDefinitionsButImplHasDefinitions() {
+        // not using swagger.json to avoid test failure because of issue 26 (where design api-doc is not allowed to have less operations under the same path)
```

Comment:
i created another [PR](https://github.com/RobWin/assertj-swagger/pull/27) that resolves the issue mentioned in this comment.